.github/workflows/ansible-tox.yml

```diff
@@ -0,0 +1,56 @@
+---
+name: Ansible tox
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+  merge_group:
+    branches:
+      - main
+    types:
+      - checks_requested
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+env:
+  TOX_ENV: qemu-ansible-core-2.16
+
+permissions:
+  contents: read
+jobs:
+  tox:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        image:
+          - centos-9
+          - centos-10
+          - fedora-41
+          - fedora-42
+
+    steps:
+      - name: Accio /dev/kvm 🪄
```

Comment:
FTR, I'm of course happy to remove the Harry Potter reference in the final PR if you want.

Reply:
As this got a +1, I replaced it with something boring :grin: